src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java

```diff
@@ -0,0 +1,83 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * write a program called PrintDayInWord, which prints �Sunday�, �Monday�, ...
+ * �Saturday� if the int variable "day" is 0, 1, ..., 6, respectively.
+ * Otherwise, it shall print �Not a valid day�.
+ */
+
+public class PrintDayInWord {
+	private String[] daysInWeek = new String[7];
+
+	private static final Logger LOGGER = Logger.getLogger(PrintDayInWord.class.getName());
+
+	public PrintDayInWord() {
+		daysInWeek[0] = "Sunday";
+		daysInWeek[1] = "Monday";
+		daysInWeek[2] = "Tuesday";
+		daysInWeek[3] = "Wednesday";
+		daysInWeek[4] = "Thursday";
+		daysInWeek[5] = "Friday";
+		daysInWeek[6] = "Saturday";
+
+	}
+
+	public static void main(String[] args) {
+		PrintDayInWord daysInWeek = new PrintDayInWord();
```

Comment:
why is PrintDayInWord object being created inside PrintDayInWord class ??
This method should be in separate class.